Servers/swagger.yaml

```
@@ -1841,3 +2127,33 @@ components:
+        owner_id:
+          type: integer
+          example: 1
+        severity:
+          type: string
+          example: "Critical"
+        likelihood:
+          type: string
+          example: "Possible"
+        risk_level:
+          type: string
+          example: "High"
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)